hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBProfile.java

```diff
@@ -44,51 +48,92 @@ public String toString() {
     }
 
     @Override
-    public ManagedColumnFamilyOptions getColumnFamilyOptions() {
-      // Write Buffer Size -- set to 128 MB
-      final long writeBufferSize = toLong(StorageUnit.MB.toBytes(128));
+    public ManagedDBOptions getDBOptions(Path optionsPath) {
```

Comment:
This is a bad idea to add the optionsPath in DB profile. Reading from ini file should be done on the DBStoreBuilder and not on the DBProfile. I believe we can load the ini file and not load from DB profile if the ini file exists on the DBStoreBuilder side. We would have to implement the same optionsPath read logic for each and every enum which is a bad idea